config/settings/qa.yml

```diff
@@ -1,12 +1,12 @@
 gcp_api_key: please_change_me
 publish_api_url: https://qa.api.publish-teacher-training-courses.service.gov.uk
 publish_url: https://qa.publish-teacher-training-courses.service.gov.uk
-find_url: https://qa.find-postgraduate-teacher-training.service.gov.uk
+find_url: https://qa.find-teacher-training-courses.service.gov.uk/
```

Comment:
I thought the plan was to get rid of these files, since they are not used anymore? @dcyoung-dev